lib/reporting/proofing_rate_report.rb

```diff
@@ -80,8 +101,18 @@ def actual_proofing_rates(reports)
 
 # rubocop:disable Rails/Output
 if __FILE__ == $PROGRAM_NAME
-  start_date = Date.parse(ARGV.first)
-
-  puts Reporting::ProofingRateReport.new(start_date: start_date).to_csv
+  end_date = if ARGV.first.match?(/\d{4}-\d{1,2}-\d{1,2}/)
+               Date.parse(ARGV.first)
+             else
+               ActiveSupport::TimeZone['UTC'].today
+             end
+  progress = !ARGV.include?('--no-progress')
+  verbose = ARGV.include?('--verbose')
```

Comment:
Another option to simplify here would be to make the `CommandLineOptions` module have an option for `require_date: false` which would let us just reuse that and get the nice argument parsing too

I just went for quick and dirty